torch/_custom_ops.py

```diff
@@ -169,7 +169,7 @@ def impl(qualname, *, device_types=("cpu", "cuda"), func=None):
     """
 
     def inner(func):
-        custom_op = _find_custom_op(qualname)
+        custom_op = _find_custom_op(qualname, also_check_torch_library=True)
```

Comment:
Maybe we want to advertise this new feature on the doc string?

Reply:
We could!